### PR TITLE
Issue739

### PR DESCRIPTION
**Pull request by pjohans**

### CODE REVIEW

**Comment by pjohans** [comment]
Notice that issue 739 works in conjunction with:
https://github.com/ding2/ding_event/pull/17

File: scripts/ddbasic.search.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
         search.insertAfter('.pane-search-result-count');
       }
       else {
-        search.insertBefore('.view-node-search-result .search-results');
+        search.insertBefore('.view-ding-node-search');
```
Comment (kasperg) [inline scripts/ddbasic.search.js]: This change should not be a part of this pull request.

**Comment by kasperg** [comment]
@pjohans: Review complete. I have one comment. Otherwise this looks good.

**Comment by pjohans** [comment]
i reverted the misplaced commit - and synchronized Issue-branch with master

**Comment by kasperg** [comment]
Reviewed and approved. Thanks!

**Pull request merged**